app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java

```diff
@@ -223,8 +223,8 @@ public void onClick(View v) {
                         break;
                     default:
                         if (scienceLab.isConnected()) {
-                            saveAndSetData(String.valueOf(String.format(Locale.ENGLISH, "%.2f", scienceLab.getVoltage(knobMarker[knobState], 1))), getString(R.string.multimeter_voltage_unit));
-                        }
+                            Double voltage =(scienceLab.getVoltage(knobMarker[knobState],1))/2;
```

Comment:
@abhinavraj23 can you give this file a little reformatting. Then this will be good to merge!